saltgui/static/scripts/routes/page.js

```diff
@@ -380,10 +387,26 @@ class PageRoute extends Route {
     document.execCommand("copy");
   }
 
-  _runCommand(evt, targetString, commandString) {
+  _runCommand(evt, targetString, commandString, targettype) {
```

Comment:
please always use the sequence "targettype, targetvalue", in that order, and always next to each other

Reply:
👍